scripts/build.sh

```diff
@@ -12,16 +12,19 @@ cd "${CONSTRUCT_ROOT}"
 # Constructor should be latest for non-native building
 # See https://github.com/conda/constructor
 echo "***** Install constructor *****"
-conda install -y "constructor>=3.3.1" jinja2 curl libarchive -c conda-forge --override-channels
-
+conda install --yes jinja2 curl libarchive --channel conda-forge --override-channels
+conda install --yes "constructor=3.3.1" --only-deps --channel conda-forge --override-channels 
 if [[ "$(uname)" == "Darwin" ]]; then
-    conda install -y coreutils -c conda-forge --override-channels
+    conda install --yes coreutils --channel conda-forge --override-channels
 fi
+
 # shellcheck disable=SC2154
 if [[ "${TARGET_PLATFORM}" == win-* ]]; then
-    conda install -y "nsis=3.01" -c conda-forge --override-channels
+    conda install -y "nsis=3.01" --channel conda-forge --override-channels
 fi
-# pip install git+git://github.com/conda/constructor@3.3.1#egg=constructor --force --no-deps
+
+constructor_rev=3.3.1_patch_for_cache
```

Comment:
If the tests pass, can you change this to a commit instead of a branch?

Reply:
I was going to patch conda forges version with the two patches.